app/jobs/client_login_request_job.rb

```diff
@@ -0,0 +1,34 @@
+class ClientLoginRequestJob < ApplicationJob
+  def perform(email_address: nil, phone_number: nil)
+    raise StandardError("No contact info provided") unless email_address.present? || phone_number.present?
+
+
```

Comment:
Another approach is to skip this rate-limit now, which is fine with me, and hopefully we can come back to it.